coopr-ui/app/features/clusters/controllers/detail-ctrl.js

```diff
@@ -78,18 +78,35 @@ function ($scope, MYSERVICEPICKER_EVENT, CrudFormBase, $state, $modal, $alert, m
       maxDate = moment.min(maxDate, moment().add(ld.step, 'ms'));
     }
     $scope.leaseMaxMs = maxDate.valueOf() - Date.now();
+    $scope.leaseMaxHumanized = myHelpers.millisecondsToString($scope.leaseMaxMs);
```

Comment:
`leaseMaxHumanized` is not used in template, could we just construct it on the fly in `showLeaseAlert()` ?